Source/CombatExtended/Harmony/Compatibility/Harmony-RunAndGun.cs

```diff
@@ -0,0 +1,95 @@
+﻿using Harmony;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Verse;
+using System.Reflection.Emit;
+using System;
+
+
+namespace CombatExtended.Harmony.Compatibility
+{
+    [HarmonyPatch]
+    class Harmony_Compat_RunAndGun
+    {
+        static readonly string logPrefix = Assembly.GetExecutingAssembly().GetName().Name + " :: ";
+        static readonly Assembly ass = AppDomain.CurrentDomain.GetAssemblies().
```

Comment:
I approve of this variable naming

Reply:
It's one of my favorite things about writing these patches targeting unreferenced assemblies, honestly.